VueJS/components/List.js

```diff
@@ -1 +1,17 @@
-export default Vue.component('list', {})
+export default Vue.component('list', {
+	props: ['todos'],
+	template: `
+		<div>
+		<ul>
+			<list-item
+				v-for="(item, index) in todos"
+				v-bind:todo="item"
+				v-bind:key="item.id"
+				v-on:delete-todo="$emit('delete-todo',index)"
```

Comment:
```suggestion
				@delete-todo="$emit('delete-todo',index)"
```
Same, if you want, you can abbreviate here.